content/Solutions/Optimizations/Networking.md

```diff
@@ -0,0 +1,339 @@
+In contrast, LACP operates at the link layer and aggregates physical links into a single logical link.
+While it provides load balancing, the granularity is typically based on source and destination IP addresses or MAC addresses, which may not distribute traffic as dynamically.
+
+SMB Multichannel allows for flexibility in adapting to varying bandwidth requirements of different applications or workloads.
+It can dynamically adjust the number of connections based on the available network resources and the demands of the data being transferred.
+
+SMB Multichannel does not require specific switch configurations or support for link aggregation, making it potentially easier to deploy in environments where switch compatibility or configuration limitations exist.
+LACP, on the other hand, relies on switch support for link aggregation.
+While it is a standardized protocol, ensuring consistent switch compatibility and configuration can sometimes be more complex in diverse network environments.
+
+SMB Multichannel is application-aware and can optimize connections based on the requirements of the specific application.
+This can lead to more efficient use of available network resources for applications that can benefit from parallel connections.
+LACP, being a lower-layer protocol, is less aware of the applications running over the network and may not optimize connections at the application layer to the same extent.
+
+These advantages can be more pronounced at higher network speeds, such as in 100G environments ([see below](#link-aggregation)).
+
+See [Setting Up SMB Multichannel]({{< relref "smbmultichannel.md" >}}) for more information.
+
+## Network Traffic Segmentation
+
+Segregate network traffic using both Layer 2 and Layer 3 methods for improved efficiency and security.
+This helps optimize bandwidth, reduce congestion, and enhance overall network resilience against potential security threats.
+
+### Layer 2 Isolation
+
+Use VLANs (Virtual Local Area Networks) to segment your network and isolate traffic.
+VLANs allow you to create logical segments within a physical network switch, even if devices in those segments are physically connected to the same switch.
+
+VLANs provide logical segmentation, allowing different groups of devices to be in separate broadcast domains regardless of their physical location in the network.
+Devices in one VLAN do not see the broadcast traffic of devices in other VLANs, reducing broadcast domain size and improving network efficiency.
+VLANs provide flexibility in network design, providing enhanced isolation, access control, and scalability.
+They are often used to group devices based on functional roles or department and optimize performance by prioritizing discrete traffic types on each VLAN.
+
+See also [Setting Up a Network VLAN]({{< relref "settingupvlan.md" >}}).
+
+#### Spanning Tree Protocol
+
```

Comment:
This section should include something along the lines that "TrueNAS does not participate in STP and does not transmit BPDUs. Because of this, it is recommended to not have STP enabled on the interfaces for the TrueNAS." Maybe also include a bit like if STP is an environmental requirement, portfast or edge ports should be used.  This is particularly relevant on HA system.
https://www.cisco.com/c/en/us/td/docs/switches/lan/catalyst4000/8-2glx/configuration/guide/stp_enha.html#wp1046787

https://www.cisco.com/c/en/us/support/docs/lan-switching/spanning-tree-protocol/24062-146.html#anc13